.readthedocs.yaml

```diff
@@ -4,6 +4,16 @@ build:
     os: ubuntu-22.04
     tools:
         python: "3"
+    commands:
+      # These come from the original RTD build command:
+      - python -mvirtualenv $READTHEDOCS_VIRTUALENV_PATH
+      - python -m pip install --upgrade --no-cache-dir pip setuptools
+      - python -m pip install --upgrade --no-cache-dir sphinx readthedocs-sphinx-ext
+      - python -m pip install --upgrade --upgrade-strategy only-if-needed --no-cache-dir .[docs]
```

Comment:
I would just put missing deps to [docs].
Since make docs below is running hatch and `python -m sphinx` is commented out, what is the purpose of installing virtualenv module and installing .[docs] like this?

Reply:
This change comes from https://github.com/teemtee/tmt/pull/2441/, let's move there.